src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf

```diff
@@ -334,6 +334,10 @@
                 <source>This value should be valid JSON.</source>
                 <target>Cette valeur doit être un JSON valide.</target>
             </trans-unit>
+            <trans-unit id="92">
+                <source>This value is not a valid timezone.</source>
+                <target>Cette valeur n'est pas un identifiant de fuseau horaire valide.</target>
```

Comment:
`Cette valeur n'est pas un fuseau horaire valide` ?
Into other translation, you do not speak about identifier. Just in French

Reply:
Fixed. thanks.